numpy/core/shape_base.py

```diff
@@ -275,3 +278,67 @@ def hstack(tup):
         return _nx.concatenate(arrs, 0)
     else:
         return _nx.concatenate(arrs, 1)
+
+def stack(arrays, axis=0):
+    """
+    Join a sequence of arrays along a new axis.
```

Comment:
I think the missing explanation is that the `axis` argument refers to the axis position in the *result* array, not in the input arrays.